build.yaml

```diff
@@ -1,14 +1,11 @@
 builders:
   built_html:
     import: 'package:built_html/builder.dart'
-    builder_factories:
-      - createBuilder
+    builder_factories: ['createBuilder']
+    build_to: cache
     build_extensions:
-      '.template.html':
-        - '.html'
+      '.template.html': ['.html']
     auto_apply: dependents
-    is_optional: false
-    build_to: cache
     required_inputs:
-      - '.js'
-      - '.css'
+    - '.js'
```

Comment:
nit: if changing to the `[]` syntax lets change it here as well

Reply:
Sure.